pkgs/development/tools/misc/unixbench/default.nix

```diff
@@ -0,0 +1,71 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchpatch
+, makeWrapper
+, perl
+, bash
+, nettools
+, glibc
+, coreutils
+}:
+
+stdenv.mkDerivation {
+  pname = "unixbench";
+  version = "unstable-2023-02-27";
+
+  src = fetchFromGitHub {
+    owner = "kdlucas";
+    repo = "byte-unixbench";
+    rev = "81e9de58c50566da8ae45c0ea7a55e7a7066ea75";
+    hash = "sha256-mAgcwvfYDrPYqrjTzlRBYlLo7DK93CPi2iavB5+oagk=";
+  };
+
+  patches = [
+    ./no_make_in_runtime.patch
+  ];
+
+  sourceRoot = "source/UnixBench";
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  buildInputs = [ perl ];
```

Comment:
```suggestion
  nativeBuildInputs = [ perl makeWrapper ];
```

Reply:
Perl exists there to tell `patchShebang` the correct path to use. Placing it in nativeBuildInputs does not work.